src/main/java/com/stripe/param/FileCreateParams.java

```diff
@@ -14,6 +14,15 @@
   @SerializedName("expand")
   List<String> expand;
 
+  /**
```

Comment:
Only extra params are added here.
I removed the part where generated code removed custom handling of `File` or `InputStream` object